backoff.py

```diff
@@ -321,6 +336,16 @@ def _handlers(hdlr, default=None):
     return defaults + [hdlr]
 
 
+# Evaluate arg that can be either a fixed value or a callable.
+def _maybe_call(f, *args, **kwargs):
```

Comment:
i would prefer to not rely on exceptions for processing and do something along the lines of http://stackoverflow.com/questions/624926/how-to-detect-whether-a-python-variable-is-a-function